crypto/multisig/wire.go

```diff
@@ -10,15 +10,15 @@ import (
 // TODO: Figure out API for others to either add their own pubkey types, or
 // to make verify / marshal accept a cdc.
 const (
-	ThresholdPubkeyAminoRoute = "tendermint/PubkeyMultisigThreshold"
+	PubKeyMultisigThresholdAminoRoute = "tendermint/PubKeyMultisigThreshold"
```

Comment:
Should we omit the multisig from the varname since it's in the package? Also should we omit the pubkey from the var name since there's no privkey

Reply:
:+1: to both.

Reply:
Mmmm .... I was hoping to reach parity between the var name and the Amino route. We can't remove `Multisig` from the amino route because there's no pkg information there, it needs to be the full `tendermint/PubKeyMultisigThreshold`.

We could consider removing `Multisig` from the var name, but I think we should keep PubKey for consistency/clarity